tools/debian/cockpit-ws.install

```diff
@@ -1,13 +1,15 @@
 etc/cockpit/ws-certs.d
 etc/pam.d/cockpit
 lib/systemd/system/cockpit.service
+lib/systemd/system/cockpit-issue.service
```

Comment:
This will ship the service, but not enable it by default. I think this is correct for now - people or OS builders can enable it manually once that new agetty lands in Debian. Also, at that point we might consider just shipping a static enablement symlink and drop the `[Install]` section.